ignite-generator/src/app/index.es

```diff
@@ -133,14 +133,12 @@ export class AppGenerator extends Generators.Base {
     this.spinner.text = status
     this.spinner.start()
     if (!isCommandInstalled('react-native')) {
-      this.log(`${xmark} Missing react-native - 'npm install -g react-native-cli'`)
-      process.exit(1)
+      this._logAndExit(`${xmark} Missing react-native - 'npm install -g react-native-cli'`)
     }
 
     // verify 1.x or higher (we need react-native link)
-    if (!Shell.exec("react-native -v | grep 'react-native-cli: [1-9]\d*\.\d\.\d'")) {
-      this.log(`${xmark} Must have at least version 1.x - 'npm install -g react-native-cli'`)
-      process.exit(1)
+    if (Shell.exec("react-native -v | grep 'react-native-cli: [1-9]\\d*\\.\\d\\.\\d'", {silent: true}).code > 0) {
```

Comment:
ahhh, I can never remember what these things return.   Great catch, and I love the `_logAndExit`!

If we open this code up again, I'm thinking we should check `!== 0` since I've heard lore of people returning `-1` but I know grep doesn't.

Great fixes!